src/Google.Storage.V1/PredefinedAcl.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Google.Storage.V1
+{
+    /// <summary>
+    /// Predefined access control lists (ACLs) that can be specified when creating or updating resources.
+    /// </summary>
+    public enum PredefinedAcl
```

Comment:
This enum doesn't have the same contents as PredefinedAclEnum defined here: https://github.com/google/google-api-dotnet-client/blob/master/Src/Generated/Google.Apis.Storage.v1/Google.Apis.Storage.v1.cs#L1008
Do PredefinedAcl and PredefinedDefaultObjectAcl need separate definitions?